.github/workflows/playwright.yml

```diff
@@ -129,13 +136,14 @@ jobs:
         env:
           PLUGIN_ID: ${{ inputs.id }}
           PLUGIN_VERSION: ${{ inputs.version }}
+        working-directory: ${{ inputs.plugin-directory }}
 
       - name: Start Grafana
         # add the -f argument only if "inputs.docker-compose-file" is defined
         run: |
           GRAFANA_VERSION=${{ matrix.GRAFANA_IMAGE.VERSION }} GRAFANA_IMAGE=${{ matrix.GRAFANA_IMAGE.NAME }} docker compose ${DOCKER_COMPOSE_FILE:+-f "$DOCKER_COMPOSE_FILE"} up -d
         env:
-          DOCKER_COMPOSE_FILE: ${{ inputs.docker-compose-file }}
+          DOCKER_COMPOSE_FILE: ${{ inputs.plugin-directory }}/${{ inputs.docker-compose-file }}
```

Comment:
This causes problems if `inputs.plugin-directory` is empty. The `docker compose` command above has some logic to omit `-f ...` if `DOCKER_COMPOSE_FILE` is empty, but now it's never empty (it's `./` instead).

Can you please cherrypick this to fix it: https://github.com/grafana/plugin-ci-workflows/commit/09b7dc4a5197ed44d52e095c2598fe12bee97fe5

Reply:
Good spot! Done, thanks :slightly_smiling_face: